frontend/components/dashboard/table/DashboardTableEmpty.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="empty delayed-fadein-animation" >
+    <span class="big_text">{{ $t('dashboard.validator.summary.empty_table') }}</span>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.empty {
+  width: 100%;
+  height: 400px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: var(--text-color-disabled);
+  gap: var(--padding);
+  cursor: pointer;
```

Comment:
remove `cursor` - this message is not clickable
and also the `gap`  is not needed as there is only one item

Reply:
done :heavy_check_mark: